src/popup.html

```diff
@@ -56,723 +50,123 @@ <h1 class="mdc-top-app-bar__title">AO3 Podfic Posting Helper</h1>
-                      <span class="mdc-deprecated-list-item__ripple"></span>
-                      <span
-                        class="mdc-deprecated-list-item__text"
-                        id="podfic-length-6-7-hours">
-                        6-7 Hours
-                      </span>
-                    </li>
-                    <li
-                      class="mdc-deprecated-list-item"
-                      aria-selected="false"
-                      data-value="7-10 Hours"
-                      role="option"
-                      data-mdc-auto-init="MDCRipple">
-                      <span class="mdc-deprecated-list-item__ripple"></span>
-                      <span
-                        class="mdc-deprecated-list-item__text"
-                        id="podfic-length-7-10-hours">
-                        7-10 Hours
-                      </span>
-                    </li>
-                    <li
-                      class="mdc-deprecated-list-item"
-                      aria-selected="false"
-                      data-value="10-15 Hours"
-                      role="option"
-                      data-mdc-auto-init="MDCRipple">
-                      <span class="mdc-deprecated-list-item__ripple"></span>
-                      <span
-                        class="mdc-deprecated-list-item__text"
-                        id="podfic-length-10-15-hours">
-                        10-15 Hours
-                      </span>
-                    </li>
-                    <li
-                      class="mdc-deprecated-list-item"
-                      aria-selected="false"
-                      data-value="15-20 Hours"
-                      role="option"
-                      data-mdc-auto-init="MDCRipple">
-                      <span class="mdc-deprecated-list-item__ripple"></span>
-                      <span
-                        class="mdc-deprecated-list-item__text"
-                        id="podfic-length-15-20-hours">
-                        15-20 Hours
-                      </span>
-                    </li>
-                    <li
-                      class="mdc-deprecated-list-item"
-                      aria-selected="false"
-                      data-value="Over 20 Hours"
-                      role="option"
-                      data-mdc-auto-init="MDCRipple">
-                      <span class="mdc-deprecated-list-item__ripple"></span>
-                      <span
-                        class="mdc-deprecated-list-item__text"
-                        id="podfic-length-over-20-hours">
-                        Over 20 Hours
-                      </span>
-                    </li>
-                  </ul>
-                </div>
-              </div>
+              <label class="checkbox-container length-checkbox">
+                <md-checkbox touch-target="wrapper" id="podfic_length_label"></md-checkbox>
+                Add a "Podfic Length" tag
+              </label>
```

Comment:
I think you mentioned while working that the extension has weird behavior if no length is selected. Idk if it's just because I have a default value stored, but my dropdown currently (as in with an older version of the extension) doesn't show a none/not selected option, I'm forced to pick a value. Is that different from the behavior you described?

Reply:
It has the same behavior now